### DISCUSSION TITLE
How to get good results?

### DISCUSSION THREAD

**Discussion by matbeedotcom** [serp-ai/bark-with-voice-clone#9] [Q&A]
Once I got something that semi-resembled my voice, but it was honestly not what I'd consider a great result. Any tips to getting a good result? Any top_p, top_k or temp settings that I should consider?

**Comment by viperwasp** [comment]
How do I begin? I have a .Npz file of 4-5 seconds long. But I can't figure out what to do next? I need to turn this file into something Bark can use and then place it somewhere? Can you help give me then next step or point me to a guide? Thanks.

**Reply by farrael004** [reply]
In the same directory as the notebook there should be a folder with the following directory structure: `bark/assets/prompts`
Put the .npz file in the `prompts` folder and use the name you gave this file into the `history_prompt` field:

```python
audio_array = generate_audio(text_prompt, history_prompt="NAME OF THE FILE GOES HERE")
```

If the name of your file is speaker.npz, then it should be:

```python
audio_array = generate_audio(text_prompt, history_prompt="speaker")
```

**Comment by devinschumacher** [comment]
We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik @farrael004 @viperwasp
